distro/res/bigDenzi/appearances.xml

```diff
@@ -171,6 +171,7 @@
 		<appearance id = "CARAVEL" x = "2" y = "12"/>
 		<!-- Corpses -->
 		<appearance id = "DEAD_NATIVE" x = "1" y = "13"/>
+		<appearance id = "PUEBLO" x = "2" y = "13"/>

```

Comment:
Just like in the Original: Hunter, Farmer, pueblo, Aztecs, Inca :)

Von meinem iPhone gesendet

> Am 04.04.2016 um 21:11 schrieb Santiago Zapata <notifications@github.com>:
> 
> In distro/res/bigDenzi/appearances.xml:
> 
> > @@ -171,6 +171,7 @@
> >  		<appearance id = "CARAVEL" x = "2" y = "12"/>
> >  		<!-- Corpses -->
> >  		<appearance id = "DEAD_NATIVE" x = "1" y = "13"/>
> > +		<appearance id = "PUEBLO" x = "2" y = "13"/>
> ahhh ok, makes sense.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>